### ISSUE TITLE
Crash when going back to Florence for the first time from Monteriggioni

### ISSUE THREAD

**Issue by segevfiner** [Sergeanur/EaglePatch#4]
When going back to Florence from Monteriggioni for the first time, the game crashes with this patch installed with it's default settings when playing the district load cutscene.

I had to disable the patch, progress past this scene, and then enable it back. (Restarting the game both times)

**Comment by samahmoo96** [comment]
Figured it out. In EaglePatchAC2.ini, set both ImproveDrawDistance and ImproveShadowMapResolution to 0. One of these seems to cause the crash. It is not Uplayitems like the mod author originally thought.

After you load in and get to an autosave, feel free to set these back to 1 and relaunch.

**Comment by ajddavid452** [comment]
yeah I encountered this too when replaying this game, only difference is that this playthrough I had eaglepatch installed, I was like wtf

**Comment by Tortillas-IT** [comment]
Same issue here, thought it was a linux issue since i'm playing this game there. Btw the crashed 2 times and then no issues
